UKHO.ExternalNotificationService.API/UKHO.ExternalNotificationService.API/Services/WebhookService.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+namespace UKHO.ExternalNotificationService.API.Services
+{
+    public class WebhookService : IWebhookService
```

Comment:
This is a factory class, which is responsible for creating objects of type IEventProcesor using a common interface. Can we please rename them as `IEventProcessorFactory` & `EventProcessorFactory` or similar?

Reply:
Thanks for highlighting, I renamed it as IEventProcessorFactory & EventProcessorFactory.